docs/source/quickstart.rst

```diff
@@ -52,3 +52,22 @@ file system changes and simply print them to the console::
         observer.join()
 
 To stop the program, press Control-C.
+
+Typing
+------
+If you are using type annotations it is important to note that
+`watchdog.observers.Observer` is not actually a class; it is a variable that
+hold the "best" observer class available on your platform.
+
+In order to correctly type your own code your should use
+`watchdog.observers.api.BaseObserver`. For example:
+
+    from watchdog.observers import Observer
```

Comment:
What about moving the whole code to its own file into `docs/sources/examples/something.py`, and include it here? Like that we will have linters, and Mypy, running checks on the file to prevent any future regression/miss when something will change.

Reply:
I should do that for other codes in that README also.

Reply:
Happy to do so, but I'm not actually familiar with RST

Reply:
Hmm that seems not possible as-is.